.github/workflows/test_on_push.yml

```diff
@@ -19,6 +19,8 @@ jobs:
           cancel_others: "true"
           paths_ignore: '["**/README.md"]'
 
+# TODO: add concurrency syntax for groups and cancelling in-progress jobs
+
```

Comment:
See https://github.com/scikit-hep/vector/blob/349aca6123873bcbaa1470d411304dab4009327a/.github/workflows/ci.yml#L11-L15 for inspiration

Reply:
FIxed in https://github.com/pybamm-team/PyBaMM/pull/3264/commits/ddc0fc587bbf95000ac2d4344f83b80d5ff0f94f, I improved the `vector` one to work on pushes outside of PRs as well